plugins/analytics/public/passive_total.py

```diff
@@ -99,8 +100,12 @@ def each(self, observable: Observable):
             last_seen = datetime.strptime(record["lastSeen"], "%Y-%m-%d %H:%M:%S")
             context["first_seen"] = first_seen
             context["last_seen"] = last_seen
-
-            new = Observable.add_text(record["resolve"]).save()
+            try:
+                context["resolve"] = record["resolve"]
+                new = Observable.add_text(record["resolve"]).save()
+            except ValueError:
+                logging.error("Could not add text observable for {}".format(record))
```

Comment:
Use an f-string here

```suggestion
                logging.error(f"Could not add text observable for {record}")
```